src/ui/qgsrelationeditorconfigwidgetbase.ui

```diff
@@ -33,7 +33,7 @@
       </sizepolicy>
      </property>
      <property name="title">
-      <string>Toolbar buttons</string>
+      <string>Toolbar buttons and context menu items</string>
```

Comment:
Maybe something shorter like "Buttons and Context Menus"? Not sure there are buttons elsewhere.
Whatever... It (group box title) should be title cased according to HIG.

Reply:
Such title was previously recently changed (in 3.26) https://github.com/qgis/QGIS/pull/48285 from "Toolbar settings" to "Toolbar buttons". Anyway, there is only a context menu, so I think "Buttons and Context Menu" or "Buttons and Context Menu Items".

Reply:
I'm fine with shorter "Buttons and Context Menu" _(singular)_. IMHO it should be clear enough for users given  the context.